src/components/Autocomplete/Autocomplete.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
```

Comment:
Sorry this whole file looks new, I moved it when I added the CSS. Moving forward, I'd like to group components in folders so that they are tightly coupled with any related files, ie. CSS, .test.js files. Since Webpack allows us to import CSS directly in the component file, it will be easier to do any future housekeeping if these items are grouped in folders.